src/device/socket/vsock.rs

```diff
@@ -291,26 +364,21 @@ impl<H: Hal, T: Transport> VirtIOSocket<H, T> {
         Ok(())
     }
 
-    fn poll_and_filter_packet_from_rx_queue<F>(
+    fn poll_and_filter_packet_from_rx_queue(
```

Comment:
We can remove `_and_filter` from the name since it doesn't filter anymore.

Reply:
Done.